WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -71,6 +76,13 @@ struct CartView: View {
     }
 }
 
+private extension CartView {
+    enum Constants {
+        static let primaryFontSize: CGFloat = 40
+        static let secondaryFontSize: CGFloat = 20
```

Comment:
You can put the whole **Font** in **Constants**. 👍

Reply:
Sounds good, and keeps the view cleaner! Updated here: https://github.com/woocommerce/woocommerce-ios/pull/13219/commits/5374a2c6fc3edf5ae1a25ceae86f000c7d6a0e9e